### ISSUE TITLE
android to windows 11 file transfer not possible

### ISSUE THREAD

**Issue by mmj-the-fighter** [tfg13/LanXchange#24]
files dragged into lanxchange android app not showing in windows 11 lanxchange app.

**Comment by tfg13** [comment]
99% firewall or wifi settings

but happy to take a look, there is a button in the app to send logs :)

**Comment by mmj-the-fighter** [comment]
Yes, that was a firewall issue. I disabled firewall for some time, and could do the transfer. Thank you for quick response.
